src/actions.tex

```diff
@@ -764,38 +752,68 @@
 			\end{itemize}
 		\end{itemize}
 		\item If hum. PR has 3+ \tradepower from Key Prov. and \idea{Mercantilism} Idea, Bots gain 1\botpower fewer
-		\item When Bot collects Trade Income, move its Merchant to that Node
+		\item When Bot collects Trade Income, move its Merchant to that Node	
 	\end{itemize}
+	}
 \end{eubox}\end{dynamiccontents*}
+
 \begin{dynamiccontents*}{frameActionCards}\begin{eubox}{\fhActionCards}
-	\subsubsection*{Action Cards \normal{(p.~18-19)}}
-	\begin{itemize}
-		\item Action cost can never go below 0
-		\item Cards in hand are secret
-		\item Discard piles are public information
-		\item If deck is empty when need to draw 
+	\begin{multicols}{2}
+		\subsubsection*{Action Cards \normal{X\monarchpower + Y\ducats (p.~18-19)}}
 		\begin{itemize}
-			\item If >5 cards in discard pile, keep the top 5, shuffle others
-			\item If ≤5 cards in discard pile, shuffle all
+			\item Action cost can never go below 0
+			\item Cards in hand are secret
+			\item Discard piles are public information
+			\item If deck is empty when need to draw 
+			\begin{itemize}
+				\item If >5 cards in discard pile, keep the top 5, shuffle others
+				\item If ≤5 cards in discard pile, shuffle all
+			\end{itemize}
+			\item Papal Controller pays -1\adminpower for actions increasing \stability (p.~46)
+			\item If Orthodox and current \stability is -1/0, pay -1\adminpower for actions increasing \stability (p.~38)
 		\end{itemize}
-	\end{itemize}
-	\smallheading{Display cards (\displaycard)}
-	\begin{itemize}
-		\item Max 2 in play per player (must be diff.)
-		\item If already having 2 and playing another
+		\smallheading{Covert Actions (\covertaction)}
 		\begin{itemize}
-			\item Must discard one in play
+			\item Can be countered by \action{Counterespionage}
+			\item When countered, pay cost and \displaycard uses 
 		\end{itemize}
-	\end{itemize}
-	\smallheading{Covert Actions (\covertaction)}
-	\begin{itemize}
-		\item Can be countered by \action{Counterespion.}
-		\item When countered
+		\smallheading{Display cards (\displaycard)}
+		\begin{itemize}
+			\item Max 2 in play per player (must be diff.)
+			\begin{itemize}
+				\item Must discard 1 of them to play another
+			\end{itemize}
+			\item Fully charge the card with \cubes when played
+			\item 1+ charges are exhausted per use (in addition to \monarchpower cost)
+		\end{itemize}
+		\smallheading{Reactions (\reaction)}
 		\begin{itemize}
-			\item Cost must be payed
-			\item Display Card uses must be spent
+			\item May be played outside your Action Turn, under specific circumstances
 		\end{itemize}
-	\end{itemize}
+		\smallheading{Card specific rules}
+		\begin{itemize}
+			\item \action{One Step Ahead}: Cannot be played on an Army with \artillery\xspace > \xspace\cavalry (p.~24)
```

Comment:
```suggestion
			\item \action{One Step Ahead}: Cannot be played on an Army with \artillery\xspace > \cavalry (p.~24)
```
Two spaces were rendered with both regular space and `\xspace` before cavalry icon